frontend/package.json

```diff
@@ -3,18 +3,28 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^4.10.2",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.4.0",
     "@testing-library/user-event": "^7.2.1",
+    "@trendmicro/react-sidenav": "^0.5.0",
+    "axios": "^0.19.2",
     "bootstrap": "^4.4.1",
+    "cypress": "^4.6.0",
```

Comment:
deveria ser uma dev-dependency. Só precisas do cypress para correr os testes e não quando o código corre em produção. 

**"dependencies"**: Packages required by your application in production.
**"devDependencies"**: Packages that are only needed for local development and testing.

https://docs.npmjs.com/specifying-dependencies-and-devdependencies-in-a-package-json-file